include/GlobalMuonTrack.h

```diff
@@ -65,6 +86,16 @@ class GlobalMuonTrack : public o2::track::TrackParCovFwd {
   bool mCloseMatch = false;
 
   SMatrix5 mResiduals2Cov;
+
+  /// for performance studies: store parameters and covariances for the MCH and
+  /// MFT
+  SMatrix5 mParametersMCH{}; ///< \brief Track parameters, MCH part of the track
+  SMatrix5 mParametersMFT{}; ///< \brief Track parameters, MFT part of the track
+  SMatrix55
+      mCovariancesMCH{}; ///< \brief Covariance matrix of track parameters, MCH
+  SMatrix55
+      mCovariancesMFT{}; ///< \brief Covariance matrix of track parameters, MFT
+  int mMCHTrackID = -1;
```

Comment:
Perhaps this should be moved to a derived class such as GlobalMuonTrackExt to be used for performance studies while we keep the neater version of GlobalMuonTrack ready to O2 reconstruction data format. Given that the analysis logic between the default and the `--matchSaveAll` mode is very different, having different classes ensures someone does not mix the methods on a macro.  Perhaps `--matchSaveAll` should be the default method at this stage. Let's discuss this on the next meeting...

Reply:
OK. Next Tuesday I will not participate to the WP9 meeting... Maybe this can wait, let's see what Kenta & Ren have done with the code.